main.tf

```diff
@@ -103,7 +113,7 @@ resource "aws_eip" "default" {
 
 # Apply the provisioner module for this resource
 module "ansible" {
-  source    = "git::https://github.com/cloudposse/tf_ansible.git?ref=tags/0.2.0"
+  source    = "git::https://github.com/cloudposse/tf_ansible.git"
```

Comment:
Why do you remove `?ref=tags/0.2.0`? If this tag do not corresponds to the latest commit in master let me know,  I can add new tag.